ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -286,12 +296,39 @@ - (void)registerExitingAncestors:(UIView *)child exitingSubviewsCount:(int)exiti
 - (void)maybeDropAncestors:(UIView *)child
 {
   UIView *parent = child.superview;
-  while (parent != nil && ![parent isKindOfClass:[RCTRootView class]]) {
+  NSNumber *parentTag = _exitingParentTags[child.reactTag];
+  [_exitingParentTags removeObjectForKey:child.reactTag];
+
+  UIViewController *childController = child.reactViewController;
```

Comment:
In the next line, in the loop condition, we compare it to `parent.reactViewController`.
The idea is to only traverse the tree up while we're still contained in the same `ViewController`, since they're the boundary between separate screens.

Reply:
Oh no, it's not. I looked at the wrong place.

Reply:
Yeah, we can get rid of it here.